src/styles/base.scss

```diff
@@ -107,16 +107,6 @@ body {
   height: 3px;
 }
 
-::-webkit-scrollbar-track-piece {
```

Comment:
Can you please let us know why this is removed ?

Reply:
Also please make practice of adding some comment on the PR itself, so that we can understand why something is removed or added easily.

Reply:
@Shekharrajak this has been removed as it was creating unnecessary extra scrollbar in the mat-tab element.

Reply:
Is it fine now in firefox or safari, IE?

Reply:
yes @Shekharrajak. I verified it for firefox and it's working fine in there.